src/nag-logger.ts

```diff
@@ -18,6 +18,7 @@ import {
  * @param nagPackName The name of the NagPack that the rule belongs to.
  * @param resource The resource the suppression is applied to.
  * @param ruleId Why the rule was triggered.
+ * @param ruleName Original name of the rule.
  * @param ruleInfo The id of the rule to ignore.
```

Comment:
I know this isn't a change you made, but let's fix this in this PR 😄

```suggestion
 * @param ruleInfo Why the rule was triggered.
```